include/config/keybind.h

```diff
@@ -11,8 +11,8 @@ struct keybind {
 	uint32_t modifiers;
 	xkb_keysym_t *keysyms;
 	size_t keysyms_len;
-	struct wl_list actions;
-	struct wl_list link;
+	struct wl_list actions;  /* action::link */
```

Comment:
I think that for `wl_list actions` is obvious what it's a list of.... a list of actions.
So I think the only `struct wl_list` entries where it's useful to have a comment is the `struct wl_list link` because it's not obvious what list is using that link... so the line below `/* rcxml::keybinds */` is where it's useful.
Do you agree?

Reply:
If i think about being new to the codebase and seeing `wl_list actions` I'd also assume there are some actions in that list. But I wouldn't necessarily know that there is a `struct action {` I could grep for in `include`.
As comments are free I'd prefer to have `/* struct action.link */` in there.

Reply:
Yes, good call. Simple thing to help readability.